### PR TITLE
Add documentation for truthiness

### PR DESCRIPTION
**Pull request by imbstack**
As it says on the tin.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -13,8 +13,8 @@
 </head>
 <body>
 <div id="root"></div>
-<script src="./runtime.40bf97ad7290d3670ddc.js" type="text/javascript"></script>
```
Comment (djmitche) [inline docs/index.html]: *grump*

Comment (imbstack) [inline docs/index.html]: I think we can make it not generate hashes after the assets? @eliperelman knows more obviously.

Comment (djmitche) [inline docs/index.html]: #117 should help.

Comment (eliperelman) [inline docs/index.html]: We need hashes for cache-busting; without the caches you could be serving outdated cached content to the browser.

Comment (eliperelman) [inline docs/index.html]: But yeah, checking in Neutrino-built assets is eww.

File: README.md [hunk 2]
```diff
@@ -116,6 +116,18 @@ template: {$json: [a, b, {$eval: 'a+b'}, 4]}
...
+### Truthiness
+
+Many values can be evaluated in context where booleans are required,
+not just booleans themselves. JSON-e defines the following values as false.
+Anything else will be true.
+
+```yaml
+context: {a: null, b: [], c: {}, d: "", e: 0, f: false}
+template: {$if: 'a || b || c || d || e || f', then: "uh oh", else: "falsey" }
+result: "falsey"
```
Comment (eliperelman) [inline README.md]: The correct spelling is "falsy" ("falsiness").

File: README.md [hunk 3]
```diff
@@ -317,7 +317,7 @@ of the context.
 * `max(a, b, ..)` -- the largest of the arguments
 * `sqrt(x)`, `ceil(x)`, `floor(x)`, `abs(x)` -- mathematical functions
 * `lowercase(s)`, `uppercase(s)` -- convert string case
-* `str(x)` -- convert string, number, boolean, or array to string
+* `str(x)` -- convert string, number, goolean, or array to string
```
Comment (djmitche) [inline README.md]: This was *not* a correct fix :)

Comment (imbstack) [inline README.md]: ... How did I even do that?

**Comment by imbstack** [comment]
I'm just going to land this since in the new world I've created, the tests will always fail. I'm going to do #117 now so that we can get tests passing again.

**Pull request merged**